### ISSUE TITLE
ion-select with no-padding attribute in RTL mode not working

### ISSUE THREAD

**Issue by snirs90** [ionic-team/ionic#11914]
**Ionic version:**  (check one with "x")
[ ] **1.x** (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[x] **3.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
ion-select with no-padding attribute in RTL mode are not reset the padding.
it is overrided by the css class:
```
[dir="rtl"] .select-md {
    padding-right: 16px;
    padding-left: 8px;
}
```

**Expected behavior:**
no-padding should override with reset the padding properties.

**Steps to reproduce:**
1. make the app RTL mode.
2. add ion-select
3. put no-padding attribute
4. The padding is not getting reset and get overrides by `[dir="rtl"] .select-md` css class.

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

global packages:

    @ionic/cli-utils : 1.3.0
    Cordova CLI      : 7.0.1 
    Ionic CLI        : 3.3.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.3.0
    @ionic/cli-plugin-ionic-angular : 1.2.0
    Cordova Platforms               : android 6.2.3
    Ionic Framework                 : ionic-angular 3.3.0

System:

    Node       : v7.9.0
    OS         : Linux 3.13
    Xcode      : not installed
    ios-deploy : not installed
    ios-sim    : not installed

**Comment by snirs90** [comment]
Also I've saw that no matter on which ion-element (at least few ion elements I've needed) I put `no-padding` attribute - it got override by some `html[dir="rtl"] ...` css class.
for example it happens for `ion-item` also.

**Comment by AmitMY** [comment]
That is "intended" behavior.

- Previous to 3.3.0, RTL support was slim, so step 1 was to add all of the `[dir="rtl"]` selectors correctly wherever needed - see https://github.com/ionic-team/ionic/pull/11342
- Step 2 - which will probably not get merged into 3.4.0, but more likely to a later version, is to handle cases like you encountered, by adding `[dir="ltr"]` as well, see https://github.com/ionic-team/ionic/pull/11635 (which fixes your issue)

Reason: Step 1 does not effect anything for ltr users, step 2 changes the entire css bundle.

There are many more RTL fixes coming soonish.

**Comment by brandyscarney** [comment]
Fixed by #11635, thanks @AmitMY :smile:
